quinn/src/lib.rs

```diff
@@ -161,8 +159,8 @@ struct Pending {
     connecting: Option<oneshot::Sender<Option<ConnectionError>>>,
     uni_opening: VecDeque<oneshot::Sender<Result<StreamId, ConnectionError>>>,
     bi_opening: VecDeque<oneshot::Sender<Result<StreamId, ConnectionError>>>,
-    cancel_loss_detect: Option<oneshot::Sender<()>>,
-    cancel_idle: Option<oneshot::Sender<()>>,
+    cancel_loss_detect: Option<TimerKey>,
+    cancel_idle: Option<TimerKey>,
```

Comment:
These names should probably be updated to reflect that they're no longer cancellation handles, e.g. to `loss_detect_timer` and `idle_timer`.